src/import_whatsapp.js

```diff
@@ -320,6 +321,7 @@ const processText = async (text) => {
 	const groupName = groupNameMatches ? groupNameMatches[1] : null;
 
 	// Check the first 3 characters to determine the format; iOS and Android
+	// iOS starts with a [ around the datetime, whereas android doesn't
```

Comment:
It'd probably make more sense to include this comment in the setImgMsgRegex function's description.